sematic/ui/src/components/NotesPanel.tsx

```diff
@@ -121,6 +124,7 @@ export default function NotesPanel(props: { rootRun: Run; selectedRun: Run }) {
                   note={note}
                   key={idx}
                   author={authorsByEmail.get(note.author_id) || anonymousUser}
+                  rootRun={rootRun}
```

Comment:
Why does the note view need a reference to the root run? Is it just so you can grey out the panel when it's not the root?

Reply:
Yes, just to have some way of signaling that a note was posted on the current or another run.